documentation/modules/ref-storage-persistent.adoc

```diff
@@ -107,10 +107,28 @@ spec:
         - broker: 2
           class: my-storage-class-zone-1c
   # ...
+  zookeeper:
+    replicas: 3
+    storage:
+      deleteClaim: true
+      size: 100Gi
+      type: persistent-claim
+      class: my-storage-class
+      overrides:
+        - broker: 0
+          class: my-storage-class-zone-1a
+        - broker: 1
+          class: my-storage-class-zone-1b
+        - broker: 2
+          class: my-storage-class-zone-1c
+  # ...
 ----
 
-As a result of the configured `overrides` property, the broker volumes use the following storage classes:
+As a result of the configured `overrides` property, the volumes use the following storage classes:
 
+* The persistent volumes of ZooKeeper node 0 will use `my-storage-class-zone-1a`.
+* The persistent volumes of ZooKeeper node 1 will use `my-storage-class-zone-1b`.
+* The persistent volumes of ZooKeeepr node 2 will use `my-storage-class-zone-1c`.
 * The persistent volumes of broker 0 will use `my-storage-class-zone-1a`.
 * The persistent volumes of broker 1 will use `my-storage-class-zone-1b`.
 * The persistent volumes of broker 2 will use `my-storage-class-zone-1c`.
```

Comment:
maybe we could specify "Kafka broker" here as we are specifying "ZooKeeper node"